src/main/kotlin/compiler/Coder.kt

```diff
@@ -182,6 +183,13 @@ class Coder(val ast: Node) {
                     value(args[0].value!!)
                 }
 
+                // O_FUNCALL O_DISCARD => O_FUNVOKE
```

Comment:
Opcode optimization for functions invoked in a statement context (such as forks).